src/ls/wsjsapi.js

```diff
@@ -508,7 +509,7 @@ var Exasol = function(url, user, pass, onconnect, onerror) {
                                      "clientName": "EXAJS",
                                      "driverName": "WS",
                                      "clientOs": "Browser",
```

Comment:
could we get a real OS here using the same `os` package?

Reply:
Sure.

We have https://nodejs.org/api/os.html#os_os_type, which returns "Linux", "Darwin", or "Windows_NT", 

or https://nodejs.org/api/os.html#os_os_platform, which returns 'aix', 'darwin', 'freebsd', 'linux', 'openbsd', 'sunos', or 'win32' (so mostly the same thing)

`os.version` is the most comprehensive (`Darwin Kernel Version 19.6.0: Tue Jan 12 22:13:05 PST 2021; root:xnu-6153.141.16~1/RELEASE_X86_64`), but possibly not very useful.

Reply:
I guess first one seems closest to what the other clients are sending.